crates/cibse_guide_e/src/chapter_7/equation_7_9/openfire_runner.rs

```diff
@@ -0,0 +1,215 @@
+pub mod integration_tests;
+
+use framework::method::calculation::Calculation;
+use framework::method::calculation::CalculationComponent;
+use framework::method::form::{Form, FormStep};
+use framework::method::parameter::ArcParameter;
+use framework::method::parameter::ParameterTrait;
+use framework::method::parameter::ParameterValue;
+use framework::method::parameter::builder::ParamBuilder;
+use framework::method::parameters::Parameters;
+use framework::method::runner::MethodRunner;
+use framework::method::tag::Tag;
+use framework::method::test::IntegrationTests;
+use framework::method::validation::ParameterError;
+use framework::method::{Method, step::Step};
+
+use crate::CIBSEGuideE;
+
+use std::sync::{Arc, RwLock};
+use std::vec;
+
+struct Symbols {
+    n_in: &'static str,
+    t: &'static str,
+    w_e: &'static str,
+    p: &'static str,
+    a: &'static str,
+    s: &'static str,
+}
+
+const SYMBOLS: Symbols = Symbols {
+    n_in: "N_{in_{max}}",
```

Comment:
```suggestion
    n_in: "N_{in(max)}",
```
In the book max is in brackets, it is not a subscript of a subscript

Reply:
Changed in both implementations.